SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -410,23 +388,37 @@ void CMeshSolver::SetWallDistance(CGeometry *geometry, CConfig *config) {
 
 void CMeshSolver::SetMesh_Stiffness(CGeometry **geometry, CNumerics **numerics, CConfig *config){
 
-  unsigned long iElem;
+  /*--- Use the config option as an upper bound on elasticity modulus.
+   *    For RANS meshes the range of element volume or wall distance is
+   *    very large and leads to an ill-conditioned stiffness matrix.
+   *    Absolute values of elasticity modulus are not important for
+   *    mesh deformation, since linear elasticity is used and all
+   *    boundary conditions are essential (Dirichlet). ---*/
+  const su2double maxE = config->GetDeform_ElasticityMod();
```

Comment:
@rsanfer I introduced a small change to the mesh elasticity, the config option above is used as an upper bound for elasticity modulus when using wall-distance or volume based mesh stiffness (for the reason in the comment).
When using constant stiffness the option is ignored and E is 1 everywhere, this change made no difference to the existing testcases.